bims/admin.py

```diff
@@ -3,11 +3,18 @@
 from django.utils.safestring import mark_safe
 from django.contrib.gis import admin
 from django.core.mail import send_mail
+
+from django.contrib.flatpages.admin import FlatPageAdmin
+from django.contrib.flatpages.models import FlatPage
+from django.db import models
+
 from geonode.people.admin import ProfileAdmin
 from geonode.people.forms import ProfileCreationForm
 from geonode.people.models import Profile
 from ordered_model.admin import OrderedModelAdmin
 
+from ckeditor.widgets import CKEditorWidget
```

Comment:
Where did you get this from?

Reply:
I have updated setup requirements.
It is from django-ckeditor package @dimasciput